### PR TITLE
Make Cheaters Blue named

### PR DESCRIPTION
**Pull request by boomzero**
Cheating is bad (

### CODE REVIEW

File: XMOJ.user.js [hunk 1]
```diff
@@ -1741,14 +1741,16 @@ else {
...
                                         let rating = await GetRating(RowData.Username).then();
-                                        if (rating > 500) {
+                                        if (RowData.QuickSubmitCount >= 2) {
+                                            UsernameLink.className += " link-info";
+                                        } else if (rating > 500) {
                                             UsernameLink.className += " link-danger";
                                         } else if (rating >= 400) {
                                             UsernameLink.className += " link-warning";
                                         } else if (rating >= 300) {
-                                            UsernameLink.className += " link-success"
```
Comment (boomzero) [inline XMOJ.user.js:1749]: I have no idea why there wasn't a ; here

Comment (langningchen) [reply XMOJ.user.js:1749]: Maybe a forgot it.

Comment (langningchen) [reply XMOJ.user.js:1749]: Please add it for me, thanks!

**Review by langningchen — APPROVED** [review]

**Comment by langningchen** [comment]
哦，不对，有个问题，如果疑似作弊的人这样只会在当前ACM排名里面变成蓝的，而不是所有的

**Review by langningchen — COMMENTED** [review]
OK

**Comment by boomzero** [comment]
> 哦，不对，有个问题，如果疑似作弊的人这样只会在当前ACM排名里面变成蓝的，而不是所有的

This is by design, otherwise username color of the same account would be different between devices

**Pull request merged**